README.md

```diff
@@ -163,6 +163,17 @@ default["audit"] = {
 }
 ```
 
+#### Attributes
+
+You can also pass in [InSpec Attributes](http://www.anniehedgie.com/inspec-basics-10) to your audit run. You do this by defining the attributes here:
```

Comment:
I would prefer if we could reference to the offical docs https://www.inspec.io/docs/reference/profiles/